src/envo/envo-edit.owl

```diff
@@ -12617,53 +12592,51 @@ AnnotationAssertion(owl:deprecated <http://purl.obolibrary.org/obo/ENVO_00000886
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/ENVO_00000887> "wwfBiome")
 AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_00000887> "\"Large\" is ambiguous. For details on \"Large rivers\" (e.g. the Mekong river) see http://worldwildlife.org/biomes/large-river-ecosystems This class will be replaced with a less ambiguous class."^^xsd:string)
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_00000887> "large river biome"^^xsd:string)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00000887> <http://purl.obolibrary.org/obo/ENVO_01000253>)
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_00000888> (large river headwater biome)
 
-AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasOBONamespace> <http://purl.obolibrary.org/obo/ENVO_00000888> "ENVO"^^xsd:string)
-AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#id> <http://purl.obolibrary.org/obo/ENVO_00000888> "ENVO:00000888"^^xsd:string)
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00000888> "A freshwater river biome which is determined by the headwaters of a large river."@en)
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/ENVO_00000888> "wwfBiome")
 AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_00000888> "\"Large\" is ambiguous. For details on \"Large river headwaters\" (e.g. the Upper Amazon headwaters) see http://worldwildlife.org/biomes/large-river-headwater-ecosystems This class will be replaced with a less ambiguous class."^^xsd:string)
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_00000888> "large river headwater biome"^^xsd:string)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00000888> <http://purl.obolibrary.org/obo/ENVO_01000253>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00000888> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/ENVO_00000887>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00000888> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_00000153>))
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_00000889> (large river delta biome)
 
-AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasOBONamespace> <http://purl.obolibrary.org/obo/ENVO_00000889> "ENVO"^^xsd:string)
-AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#id> <http://purl.obolibrary.org/obo/ENVO_00000889> "ENVO:00000889"^^xsd:string)
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00000889> "A freshwater river biome which is determined by the delta of a large river."@en)
```

Comment:
seem like language like 'a biome hosted by the delta of a river'  would be better way to define.  Isn't a biome hosted by some environment?